v8/payloads/interactions.ts

```diff
@@ -140,7 +140,9 @@ export type APIInteractionResponsePong = InteractionResponsePayload<APIInteracti
 
 export type APIInteractionResponseAcknowledge = InteractionResponsePayload<APIInteractionResponseType.Acknowledge>;
 
-export type APIInteractionResponseAcknowledgeWithSource = InteractionResponsePayload<APIInteractionResponseType.AcknowledgeWithSource>;
+export type APIInteractionResponseAcknowledgeWithSource = InteractionResponsePayload<
+	APIInteractionResponseType.AcknowledgeWithSource
+>;
```

Comment:
eslint is failing on this but I didn't change this 🤔 probably was changed by eslint in vscode or something

Reply:
Update deps, then run lint :p